mitodl/mitxonline
# added valid mitx logo

#### Pre-Flight checklist
  
#### What are the relevant tickets?
Ticket no #164

#### What's this PR do?
This PR updates the logo in email_base.html to the one used in MITxOnline

#### How should this be manually tested?
This PR can be tested in the following ways
- Create a new user and verify email has valid logo
- Reset Password and verify email has valid logo 
- Enrol in course and verify email has valid logo

#### Any background context you want to provide?
During testing on local dev environment the image is still not visible but that is due to gmail image proxy.
It rewrites image src with its proxy server url and is unable to fetch the image from a local machine.
It should be testable on RC or any deployed instance.

#### Screenshots (if appropriate)

<img width="1040" alt="Screenshot 2021-09-13 at 1 59 23 PM" src="https://user-images.githubusercontent.com/87968618/133058365-0cf15343-30d0-43c6-99b1-93794b648028.png">

<img width="1040" alt="Screenshot 2021-09-13 at 2 26 19 PM" src="https://user-images.githubusercontent.com/87968618/133059994-19bbc042-11c1-4f1f-aafe-8c9907c4de16.png">

<img width="981" alt="Screenshot 2021-09-14 at 4 01 24 PM" src="https://user-images.githubusercontent.com/87968618/133246401-be42c55c-9e01-43b1-b1d4-48792c65b19d.png">
